src/main/webapp/js/simulations.js

```diff
@@ -15,6 +15,10 @@
 import { toQueryString } from './utils.js';
 
 window.retrieveSimulations = retrieveSimulations; // Add function to global scope.
+const ASC = 0; // Query.SortDirection.ASCENDING = 0 (according to Datastore API documentation).
```

Comment:
Relying on the ordering of enums is rather brittle; as mentioned in my comment on the servlet, prefer to just pass "ASCENDING" or "DESCENDING" as strings in the URL.